plugins/modules/helm.py

```diff
@@ -496,7 +496,8 @@ def main():
             changed = True
 
         elif force or release_values != release_status['values'] \
-                or (chart_info['name'] + '-' + chart_info['version']) != release_status["chart"]:
+                or (chart_info['name'] + '-' + chart_info['version']) != release_status["chart"] \
+                or chart_info['appVersion'] != release_status["app_version"]:
```

Comment:
If an `appVersion` is not provided in the chart, would this fail gracefully?

Reply:
Good call, the `appVersion` key is optional. I've added a second chart and associated tests.